components/autosize_textarea.tsx

```diff
@@ -12,6 +12,10 @@ type Props = {
     onHeightChange?: (height: number, maxHeight: number) => void;
     onInput?: (e: FormEvent<HTMLTextAreaElement>) => void;
     placeholder?: string;
+    style?: CSSProperties;
+    className?: string;
+    rows?: number;
+    maxLength?: number;
```

Comment:
These new props as well as most of the existing ones come from the underlying textarea element, so I wonder if there's some way to just tell it that any of those props work here?

We haven't used it anywhere yet, but I just found a `React.HTMLProps<HTMLTextAreaElement` type that we might be able to use here. Perhaps we could replace this whole type with the following?
```
type Props = React.HTMLProps<HTMLTextAreaElement> & {
    onHeightChange?: (height: number, maxHeight: number) => void;
};
```

Reply:
Seems reasonable.
I replaced the type and the build passes local testing.